node/pkg/watchers/near/poll.go

```diff
@@ -34,15 +34,24 @@ func (e *Watcher) recursivelyReadFinalizedBlocks(logger *zap.Logger, ctx context
 
 	// SECURITY: Sanity checks for the block header
 	if startBlock.Header.Hash == "" || startBlock.Header.Height == 0 || startBlock.Header.PrevBlockHash == "" {
+		logger.Debug("recursivelyReadFinalizedBlocks: json parse error")
```

Comment:
Idiomatic go says to log errors you don't return. Is it possible to log one level up where you return the error to?